Components/ArticleCard.jsx

```diff
@@ -0,0 +1,35 @@
+import Stack from 'react-bootstrap/Stack';
+
+export default function ArticleCard({articles}){
+      return (
+        <section>
+            {articles.map((article, index) => (
+                <ul className='articleList' key={[article.id, index]}>
```

Comment:
I'm slightly confused by this component,
It's name is `ArticleCard` which implies a single card, but it's mapping over the list of all articles. 
Then the map is returning an `unordered list <ul>` for each article

It feels like this component should be `ArticleList` which has a `<ul>` and a map, where the map returns an `<li> list item` for each article